Gemfile

```diff
@@ -15,3 +15,5 @@ end
 
 # https://github.com/jekyll/jekyll-redirect-from/issues/150
 Encoding.default_external = Encoding::UTF_8
+
+gem "jekyll-github-metadata", "~> 2.13"
```

Comment:
I am not as familiar with Jekyll/Ruby libraries - so perhaps it would be good to add a quick comment here to explain what this gem is for?